### ISSUE TITLE
 Execution default of goal com.jcabi:jcabi-maven-plugin:0.15.0:ajc failed. NullPointerException

### ISSUE THREAD

**Issue by slash33** [jcabi/jcabi-maven-plugin#78]
hi, with jcabi-maven-plugin version 0.15.0 i have this exception : 
 Execution default of goal com.jcabi:jcabi-maven-plugin:0.15.0:ajc failed. NullPointerException

pom.xml : 
```
		<plugin>
				<groupId>com.jcabi</groupId>
				<artifactId>jcabi-maven-plugin</artifactId>
				<version>0.15.0</version>
				<configuration>
					<source>11</source>
					<target>11</target>
				</configuration>
				<executions>
					<execution>
						<goals>
							<goal>ajc</goal>
						</goals>
					</execution>
				</executions>
				<dependencies>
					<dependency>
						<groupId>org.aspectj</groupId>
						<artifactId>aspectjtools</artifactId>
						<version>1.9.8.M1</version>
					</dependency>
					<dependency>
						<groupId>org.aspectj</groupId>
						<artifactId>aspectjweaver</artifactId>
						<version>1.9.8.M1</version>
					</dependency>
				</dependencies>
			</plugin>
```

**Comment by omax** [comment]
Same issue for me too. Details:

Caused by: java.lang.NullPointerException
    at org.apache.maven.shared.dependency.graph.internal.Maven31DependencyGraphBuilder.buildDependencyGraph (Maven31DependencyGraphBuilder.java:99)
    at org.apache.maven.shared.dependency.graph.internal.DefaultDependencyGraphBuilder.buildDependencyGraph (DefaultDependencyGraphBuilder.java:81)
    at com.jcabi.maven.plugin.AjcMojo.classpath (AjcMojo.java:328)
    at com.jcabi.maven.plugin.AjcMojo.executeAjc (AjcMojo.java:256)
    at com.jcabi.maven.plugin.AjcMojo.execute (AjcMojo.java:220)
    at org.apache.maven.plugin.DefaultBuildPluginManager.executeMojo (DefaultBuildPluginManager.java:137)

**Comment by yegor256** [comment]
@omax @slash33 fixed now, try version `0.16.0`
